src/v2/providers/https.ts

```diff
@@ -113,6 +120,7 @@ export function onRequest(
         httpsTrigger: {
           allowInsecure: false,
         },
+        invoker,
```

Comment:
Let's omit if it's not set and always make it an array if set.

Reply:
Removed here, added it to global options, and added a `convertIfPresent()` call to the change invoker to an array.